src/components/EditRouteModal/EditRouteModal.jsx

```diff
@@ -133,6 +133,7 @@ const EditRouteModal = ({
 }) => {
   const [assignedDriverId, setAssignedDriverId] = useState('');
   const [route, setRoute] = useState({});
+  const [assignedRouteName, setAssignedRouteName] = useState('');
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'assignedRouteName' is assigned a value but never used.